.github/workflows/chatops.yml

```
@@ -0,0 +1,37 @@
+name: chatops
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  default:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: "Handle common commands"
+        uses: cloudposse/actions/github/slash-command-dispatch@0.15.0
```

Do you want to use the new slash command?
```suggestion
        uses: cloudposse/actions/github/slash-command-dispatch@0.16.0
```

---

Ah, yes, but after we merge the upstream pr.